distribution/docs/src/main/resources/content/_metadataAttributes/nsili-attributes-table.adoc

```diff
@@ -0,0 +1,711 @@
+:title: NSILI (STANAG 4559) Attribute Mappings
+:type: subAppendix
+:order: 900
+:parent: Format-specific Attribute Mappings
+:status: published
+:summary: NSILI (STANAG 4559) Attribute Mappings.
+
+== {title}
+
+.NSILI (STANAG 4559) Attribute Mappings
+[cols="5" options="header"]
+|===
+
+|NSILI Entity Field
+|NSILI Attribute Field
+|Term
+|Translation
+|Type
+
+|NSIL_ALL_VIEW
+4+|_no direct fields_
+
+|NSIL_APPROVAL
+|approvedBy
+3+|_not mapped_
+
+|NSIL_APPROVAL
+|dateTimeModified
+3+|_not mapped_
+
+|NSIL_APPROVAL
+|status
+3+|_not mapped_
+
+|NSIL_ASSOCIATION
+4+|_no direct fields_
+
+|NSIL_ASSOCIATION_VIEW
+4+|_no direct fields_
+
+|NSIL_CARD
+|dateTimeModified
+|`modified` if the card is the main node
```

Comment:
@troymohl, what do you think this should say?
https://github.com/codice/alliance/blob/bbce251d655993c86647f2a7f08bb14e1423e2b1/catalog/nsili/catalog-nsili-transformer/src/main/java/org/codice/alliance/nsili/transformer/DAGConverter.java#L170-L174

Reply:
Maybe this would be better if we just left it `modified`. On `Identifier` below maybe we add a note if it was an association node it defines that associated metacard.